front/less/pages/category.less

```diff
@@ -181,8 +181,8 @@
     float: right;
     font-size: @f-size-base + 3;
     padding: 6px 10px;
-    margin-left: 5px;
-    width: 105px;
+    margin-left: 2.5%;
```

Comment:
Let's use `em` or `rem` for `margin`/`padding`
[Here](https://stackoverflow.com/a/15851071/6800156) you can read the reason for this

Reply:
@artemiy312 , thank you for the link! It's really good.

Let me quote text from given link:
> For layout type things like the sizes of boxes, you want to use %

in this css class i'm specifying exactly layout